docs/admonitions.rst

```diff
@@ -57,13 +57,33 @@ Version Directives
 ******************
 
 The `versionadded`, `versionchanged`, and `deprecated` directives defined by Sphinx are also
-rendered by the sphinx-immaterial theme as admonitions.
+overridden by the sphinx-immaterial theme to render as :rst:dir:`admonition`\ s with added
+functionality (like optionally making them :rst:`:collapsible:`).
 
-These admonition styles can be customized using the `Custom Admonitions`_ feature provided the
+Beware, these directive overrides do not support the :rst:`:title:` and
+:rst:`:no-title:` options because
+
+1. They require 1 argument (and optionally 1 additional argument) that gets used in the title.
+2. The default title cannot be changed.
+
+.. confval:: sphinx_immaterial_override_version_directives
+
+   This theme can be configured to disallow these overrides if the new features cause a conflict
+   with other Sphinx extensions.
+
+   .. code-block:: python
+      :caption: Disallow overriding builtin version directives in conf.py
+
+      sphinx_immaterial_override_version_directives = False
+
+These admonition styles can be customized (despite the
```

Comment:
perhaps "regardless of the setting of ..." instead of "despite"

Reply:
I'm not a fan of nesting prepositional phrases; "of ... of ..." may fail in translation to non-English languages. I like the "regardless" word choice though.